examples/nextjs-with-typescript/pages/MuxPlayer.tsx

```diff
@@ -131,160 +143,90 @@ const PlayerSizeHeights = {
 };
 
 const DEFAULT_INITIAL_STYLES_STATE = {
-  // width: PlayerSizeWidths[MediaChromeSizes.XS] + 'px',
-};
-
-const DEFAULT_INITIAL_OPTION_STYLES_STATE = {
```

Comment:
"option styles state" was no longer necessary with the flexbox refactor mentioned above.